weave/trace_server/trace_server_interface.py

```diff
@@ -686,6 +686,18 @@ class FeedbackCreateReq(BaseModel):
             }
         ]
     )
+    annotation_ref: Optional[str] = Field(
```

Comment:
it would be nice if we could type this to a kind of ref, like `objectRef`, with a pydantic validator and then check its construction in the client.

Reply:
Agreed